README.md

```diff
@@ -245,7 +246,7 @@ This parameter is for filtering events through the tags. You need to pass throug
 
 ex. `www.website.com/?tags=tag_one,tag_two`
 
-* *other parameters* - `?stars_on__gte , ?starts_on__lt , ?ends_on__gte , ?ends_on__lt` etc.
+* *other parameters* - `?starts_on__gte , ?starts_on__lt , ?ends_on__gte , ?ends_on__lt` etc.
```

Comment:
I'm not super familiar with wp so bear with me. Does this typo correction have any backwards compatibility concerns?